rm_vk/rm_vk/Common/CustomNavigation/CustomInteractiveTransition.swift

```diff
@@ -0,0 +1,49 @@
+// CustomInteractiveTransition.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Интерактивная анимация закрытия экрана
+final class CustomInteractiveTransition: UIPercentDrivenInteractiveTransition {
+    // MARK: - Public property
+
+    var viewController: UIViewController? {
+        didSet {
```

Comment:
```suggestion
        didSet {
```
нам скорее всего скажут это вынести в метод, но я не смогла